### PR TITLE
Prepare to release 1.13.0.rc1

### PR DESCRIPTION
**Pull request by ahuang11**

### CODE REVIEW

**Comment by hoxbro** [comment]
A bit of a chicken and age here, as you need a git tag for the Python version

I, therefore, usually do these pushes directly to the main. 

As you are likely aware, an official release does not make much sense until we release Panel 1.5, because of Bokeh 3.5 support.

File: geoviews/package-lock.json
Comment (hoxbro) [inline geoviews/package-lock.json:12]: This should be 3.5

**Comment by ahuang11** [comment]
Right I was planning to make a git tag for the Python version after this was approved + merged, and it was a good thing you picked up on the 3.5 thing

File: geoviews/package-lock.json
Comment (hoxbro) [reply geoviews/package-lock.json:12]: Have opened here: https://github.com/holoviz/geoviews/pull/740

Comment (ahuang11) [reply geoviews/package-lock.json:12]: Thanks; how did you get the other versions?

**Comment by ahuang11** [comment]
Ok I'll push this to main directly and release rc

**Comment by ahuang11** [comment]
Nevermind, I don't have permissions I think

```
Enumerating objects: 9, done.
Counting objects: 100% (9/9), done.
Delta compression using up to 12 threads
Compressing objects: 100% (5/5), done.
Writing objects: 100% (5/5), 471 bytes | 471.00 KiB/s, done.
Total 5 (delta 4), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (4/4), completed with 4 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/main.
remote: error: 7 of 7 required status checks are expected.
To https://github.com/holoviz/geoviews.git
 ! [remote rejected] main -> main (protected branch hook declined)
error: failed to push some refs to 'https://github.com/holoviz/geoviews.git'
```

**Comment by hoxbro** [comment]
Then just merge this and try to see if you can tag.

**Review by hoxbro — APPROVED** [review]

**Comment by hoxbro** [comment]
What if you click on the small arrow next to the button?

**Comment by ahuang11** [comment]
Which one? Isn't that already in the screenshot

**Comment by hoxbro** [comment]
image

**Comment by ahuang11** [comment]
Yes those are the options I see; same as the screenshot above

image

**Pull request merged**
